tests/contrib/hooks/test_grpc_hook.py

```diff
@@ -48,21 +48,21 @@ def get_airflow_connection_with_port():
 
 
 class StubClass:
-    def __init__(self, channel):
+    def __init__(self, _):
         pass
 
     def single_call(self, data):
         return data
 
-    def stream_call(self, data):
+    def stream_call(self, data):  # pylint: disable=unused-argument
         return ["streaming", "call"]
 
 
 class TestGrpcHook(unittest.TestCase):
     def setUp(self):
         self.channel_mock = mock.patch('grpc.Channel').start()
 
-    def custom_conn_func(self, connection):
+    def custom_conn_func(self, _):
```

Comment:
The `unused-argument` is quite popular in tests, so maybe we can unify the process of fixing this error. What about using `*args, **kwargs` in such cases? This will resolve the problem and preserve all logic.

Reply:
I think it's valuable to keep the signature of the function. I think that in such cases it was better to use MagicMock, but this is not the purpose of this PR. In the future, we can rewrite tests to use MagicMock much more often.